handlers/graph.go

```diff
@@ -491,18 +493,19 @@ func populateTrafficMapTcp(trafficMap graph.TrafficMap, vector *model.Vector, o
 		lSourceApp, sourceAppOk := m["source_app"]
 		lSourceVer, sourceVerOk := m["source_version"]
 		lDestSvcNs, destSvcNsOk := m["destination_service_namespace"]
-		lDestSvcName, destSvcNameOk := m["destination_service_name"]
+		lDestSvc, destSvcOk := m["destination_service_name"]
+		lDestWlNs, destWlNsOk := m["destination_workload_namespace"]
 		lDestWl, destWlOk := m["destination_workload"]
 		lDestApp, destAppOk := m["destination_app"]
 		lDestVer, destVerOk := m["destination_version"]
 
 		// TCP queries doesn't use destination_service_namespace for the unknown node.
 		// Check if this is the case and use destination_workload_namespace
-		if !destSvcNsOk {
-			lDestSvcNs, destSvcNsOk = m["destination_workload_namespace"]
-		}
+		// if !destSvcNsOk {
+		//	lDestSvcNs, destSvcNsOk = m["destination_workload_namespace"]
+		// }
```

Comment:
why is this commented out?

Reply:
Related to the above, we no longer need the special handling because we use the same groupBy for different queries.  I'll remove it.